src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Clickable } from '../Clickable';
+import { Text } from '../Text';
+import { Input } from '../Field';
+
+import './PasswordInput.scss';
+
+export type PasswordInputProps = {
+  error?: string | boolean | undefined;
+  dataTestId?: string;
+  id: string;
+  placeholder: string;
+};
```

Comment:
Let's make this accept all props of `Input` and add extra ones if needed, similar to how we're doing it for Button here https://github.com/ModaOperandi/om/blob/master/src/components/Button/Button.tsx#L7-L13